build/dependencies.yaml

```diff
@@ -87,7 +87,7 @@ dependencies:
 
   # Golang
   - name: "golang: upstream version"
-    version: 1.17.13
+    version: 1.18.7
```

Comment:
1.18.8 fixes security issues, 1.18.8 has been used to build in the 1.24 branch

https://go.dev/doc/devel/release#go1.18